test/src/unit-cppapi-config.cc

```diff
@@ -90,16 +90,24 @@ TEST_CASE(
     "C++ API: Config Environment Variables Default Override",
     "[cppapi], [cppapi-config]") {
   tiledb::Config config;
+  std::string key = "vfs.num_threads";
```

Comment:
`const` this here. i'm not going to pick on the other non-const lines below since they already exist.